OpenRA.Mods.Common/Widgets/ProductionPaletteWidget.cs

```diff
@@ -601,6 +627,18 @@ public override void Draw()
 							TextColor, Color.Black, 1);
 					}
 				}
+
+				if (CurrentQueue is BulkProductionQueue bulkProductionQueue)
+				{
+					var readyActors = bulkProductionQueue.GetPurchasedActors();
+					if (readyActors.Any(a => a.Name == icon.Name))
```

Comment:
```suggestion
					var readyActors = bulkProductionQueue.GetPurchasedActors().Count(a => a.Name == icon.Name);
					if (readyActors > 0)
```

as you only need the number count immediately

Reply:
`if (readyActors > 0)` probably also isn't necessary as Count never return negative values.